src/main/java/com/backbase/billpay/fiserv/enrolment/EnrolmentServiceImpl.java

```diff
@@ -101,4 +107,24 @@ private SubscriberEnrollRequest createEnrolmentRequest(Header header) {
                                                           .build())
             .build();
     }
+
+    @Override
+    public UserByIdGetResponseBody getSubscriberInfo(Header header, String userId) {
+        SubscriberInfoRequest request = createSubscriberInfoRequest(header);
+        LOGGER.debug("Request: {}", request);
+        SubscriberInfoResponse response = client.call(request, SOAP_ACTION_INFO);
+        LOGGER.debug("Received response: {}", response);
```

Comment:
Please remove the response object from the log, perhaps just
`LOGGER.debug("Received response for user: {}, userId);`

Reply:
Done